website/docs/components/link/inline/partials/guidelines/guidelines.md

```diff
@@ -1,3 +1,85 @@
 ## Usage
 
-Usage documentation for this component is coming soon. In the meantime, help us improve this documentation by letting us know how your team is using it in [#team-design-systems](https://hashicorp.slack.com/archives/C7KTUHNUS) (internal only).
\ No newline at end of file
+### When to use
+
+- To navigate to a new destination, either internally or externally.
+- Within a block of body text or other text element.
+
+### When not to use
+
+- As a standalone element, consider [Standalone Link](/components/link/standalone)
+- To trigger an action or event, like a form submission, consider [Button](/components/button)
+
+## Usage in Figma
+
+The Inline Link is only published as a component in Ember, there is no corresponding component in Figma. To achieve the same results, apply **text** and **color** styles from the [Foundations library](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?type=design&node-id=2130%3A2&t=6XBReWOxMRTiW1Iv-1) to your text.
+
+- Text styles can be found under text styles `Body`, `Code`, or `Display`.
```

Comment:
There's only 1 weight of the link in a Figma style, though. We should indicate that folks may need to change the weight based on the surrounding text as the weight should be inherited from the content around the link.